pkg/validation/internal/removed_apis_test.go

```diff
@@ -101,16 +173,6 @@ func TestValidateDeprecatedAPIS(t *testing.T) {
 				"Migrate the API(s) for CRD: ([\"etcdbackups.etcd.database.coreos.com\" " +
 				"\"etcdclusters.etcd.database.coreos.com\" \"etcdrestores.etcd.database.coreos.com\"])"},
 		},
-		{
-			name: "should not return a warning or error when has minKubeVersion but the k8sVersion informed is <= 1.15",
```

Comment:
shouldn't we still have a test for a no-warning case?

At least for 1.22 < k8sVersion < 1.26 to ensure no-warning for 1.26 deprecated APIs?

Reply:
That makes no sense and only brings complexities to keep it maintained.

Why should we not warn in any scenario that the project is using an API that will be or is removed in some specific version? We cannot raise an error in this case.